README.rst

```diff
@@ -24,7 +24,7 @@ Write
 .. code-block:: python
 
     >>> from sqlitedict import SqliteDict
-    >>> db = SqliteDict("/tmp/db.sqlite")
+    >>> db = SqliteDict("db.sqlite")
```

Comment:
This is a bad idea, because if the user has a `db.sqlite` in their present directory, we:

1. Write to the DB (bad but not horrible)
2. Trash the DB when the tests exit (this is quite bad)

Reply:
Agreed, I thought of that. 
How about we rename the file to "example.db"? then there's no risk of deleting anything important.

Reply:
By "user", you mean a developer running the doctest locally, right?

Reply:
Or if you mean someone copy-pasting these examples locally, I wouldn't worry about that much. Both `db.sqlite` and `example.sqlite` are fine, developers are not idiots (and if they are, we they'll run into other trouble too, nothing much we can do).

Reply:
I mean in the event that somebody runs the doctest locally. They might not expect it do delete the db.sqlite in the current directory.  (I hope a developer wouldn't run unit tests where database is sitting but who knows ¯\_(ツ)_/¯  ).